### ISSUE TITLE
Support for Legacy DFU for SDK <= 11

### ISSUE THREAD

**Issue by gugagop** [infsoft-locaware/nrfdfu#4]
Hi,
I have a question if it is possible to use this library for a Serial DFU using SDK11 serial bootloader.

**Comment by br101** [comment]
Hi, I assume it should work, but I have only tested it with serial and SDK 15 and 16 bootloaders.

**Comment by br101** [comment]
No, bootloaders <= SDK11 have a different protocol called LEGACY DFU. I have an experimental branch working on that, but for now I will not continue to work on it

**Comment by hongkongkiwi** [comment]
Where's the experimental branch?

**Comment by br101** [comment]
I have just pushed the legacy-dfu branch. It is heavy WIP and as I said I have no motivation to work on it now. Welcome to pick it up.
